### ISSUE TITLE
Jest: setting setupTestFrameworkScriptFile causes test reporting to stop

### ISSUE THREAD

**Issue by narthollis** [wallabyjs/public#804]
### Issue description or question

When setting an the Jest `setupTestFrameworkScriptFile` configuration value, Wallaby states that no tests have been run (No Failing test, 0 passing)

The issue appears to be similar to #763 - however that states the bug was resolved in `core 1.0.283` and my wallaby console reports I am running `core 1.0.296`

I have included my startup script - though I was able to get the bug to happen even with an empty setup test file.

### Wallaby.js configuration file

```javascript
process.env.NODE_ENV = 'test';

// eslint-disable-next-line func-names
module.exports = function (wallaby) {
    return {
        files: [
            { pattern: 'node_modules/react/dist/react-with-addons.js', instrument: false },
            'src/**/*.js*',
            '!src/**/*.test.js*',
            '!src/**/*.fixture.js',

            { pattern: 'test/*.js', instrument: false },
        ],

        tests: [
            'src/**/*.test.js',
            'src/**/*.test.jsx',
        ],

        compilers: {
            '**/*.js': wallaby.compilers.babel(),
            '**/*.jsx': wallaby.compilers.babel(),
        },

        env: {
            type: 'node',
            runner: 'node',
            params: {
                runner: '--harmony',
            },
        },

        testFramework: 'jest',

        setup: function (wallaby) {
            wallaby.testFramework.configure({
                setupTestFrameworkScriptFile: '<rootDir>/test/setup-before-after.js',
                moduleFileExtensions: ['js', 'json', 'jsx', 'node'],
                moduleNameMapper: {
                    '^.+\\.(css|scss|sass)$': '<rootDir>/test/styleMock.js',
                    '^.+\\.(gif|ttf|eot|svg)$': '<rootDir>/test/fileMock.js',
                },
                // testRegex: '/__tests__/.*|\\.(test|spec)\\.(js|jsx)$',
            });
        },
    };
};
```

### test/setup-before-after.js
```javascript
/* eslint-env jest */
// Replace console.error during tests to raise an exception, causing the tests to fail.

function mockError(err) {
    throw new Error(err);
}

/* eslint-disable no-console */
const realError = console.error;

beforeEach(() => {
    console.error = mockError;
});

afterEach(() => {
    console.error = realError;
});
/* eslint-enable no-console */
```

### Code editor or IDE name and version
WebStorm 2016.3EAP

### OS name and version
Windows 10

**Comment by ArtemGovorov** [comment]
I have set up [this sample repo](https://github.com/wallabyjs/wallaby-804) with your wallaby config file and the `setupTestFrameworkScriptFile` file and it's working for me. Could you please try the repo and, if it's working for you, could you try adding more project specific details I'm missing to reproduce the issue?

**Comment by narthollis** [comment]
After checking out that repo, I setup jest in my IDE and ran it, and i get the following output in the following images.

The issue I am experiencing is in the "Failing Tests" tab, where it shows `No failing tests, 0 passing`. In editor code coverage and test fail/succeed indicators display as expected.

![image](https://cloud.githubusercontent.com/assets/67732/18820961/206120b6-83e1-11e6-9ea9-b1cc4ec66150.png)
![image](https://cloud.githubusercontent.com/assets/67732/18820969/2d583084-83e1-11e6-8587-409eebb98009.png)

**Comment by ArtemGovorov** [comment]
Trying on Windows, but can't yet reproduce the issue. What's your node.js version (`node -v`)?

**Comment by narthollis** [comment]
Node Version: v6.5.0

Also, I just noticed i said windows 10 in my original post, but its actually windows 7.

```
> npm ls
wallaby-804@1.0.0 E:\temp\wallaby-804
+-- babel-core@6.14.0
| +-- babel-code-frame@6.11.0
| | +-- esutils@2.0.2
| | `-- js-tokens@2.0.0
| +-- babel-generator@6.14.0
| | `-- detect-indent@3.0.1
| |   +-- get-stdin@4.0.1
| |   `-- repeating@1.1.3
| |     `-- is-finite@1.0.1
| +-- babel-helpers@6.8.0
| +-- babel-messages@6.8.0
| +-- babel-register@6.14.0
| | +-- core-js@2.4.1
| | +-- home-or-tmp@1.0.0
| | | +-- os-tmpdir@1.0.1
| | | `-- user-home@1.1.1
| | +-- mkdirp@0.5.1
| | | `-- minimist@0.0.8
| | `-- source-map-support@0.2.10
| |   `-- source-map@0.1.32
| |     `-- amdefine@1.0.0
| +-- babel-runtime@6.11.6
| | `-- regenerator-runtime@0.9.5
| +-- babel-template@6.15.0
| +-- babel-traverse@6.15.0
| | +-- globals@8.18.0
| | `-- invariant@2.2.1
| |   `-- loose-envify@1.2.0
| |     `-- js-tokens@1.0.3
| +-- babel-types@6.15.0
| | `-- to-fast-properties@1.0.2
| +-- babylon@6.11.2
| +-- convert-source-map@1.3.0
| +-- debug@2.2.0
| | `-- ms@0.7.1
| +-- json5@0.4.0
| +-- lodash@4.16.1
| +-- minimatch@3.0.3
| | `-- brace-expansion@1.1.6
| |   +-- balanced-match@0.4.2
| |   `-- concat-map@0.0.1
| +-- path-exists@1.0.0
| +-- path-is-absolute@1.0.0
| +-- private@0.1.6
| +-- shebang-regex@1.0.0
| +-- slash@1.0.0
| `-- source-map@0.5.6
`-- jest-cli@15.1.1
  +-- ansi-escapes@1.4.0
  +-- callsites@2.0.0
  +-- chalk@1.1.3
  | +-- ansi-styles@2.2.1
  | +-- escape-string-regexp@1.0.5
  | +-- has-ansi@2.0.0
  | | `-- ansi-regex@2.0.0
  | +-- strip-ansi@3.0.1
  | `-- supports-color@2.0.0
  +-- graceful-fs@4.1.6
  +-- istanbul-api@1.0.0-aplha.10
  | +-- async@1.5.2
  | +-- clone@1.0.2
  | +-- fileset@0.2.1
  | | +-- glob@5.0.15
  | | | +-- inflight@1.0.5
  | | | `-- inherits@2.0.3
  | | `-- minimatch@2.0.10
  | +-- istanbul-lib-hook@1.0.0-alpha.4
  | | `-- append-transform@0.3.0
  | +-- istanbul-lib-report@1.0.0-alpha.3
  | | +-- path-parse@1.0.5
  | | +-- rimraf@2.5.4
  | | | `-- glob@7.1.0
  | | |   `-- fs.realpath@1.0.0
  | | `-- supports-color@3.1.2
  | |   `-- has-flag@1.0.0
  | +-- istanbul-lib-source-maps@1.0.1
  | +-- istanbul-reports@1.0.0-alpha.8
  | | `-- handlebars@4.0.5
  | |   +-- optimist@0.6.1
  | |   | +-- minimist@0.0.10
  | |   | `-- wordwrap@0.0.3
  | |   +-- source-map@0.4.4
  | |   `-- uglify-js@2.7.3
  | |     +-- async@0.2.10
  | |     +-- uglify-to-browserify@1.0.2
  | |     `-- yargs@3.10.0
  | |       +-- camelcase@1.2.1
  | |       +-- cliui@2.1.0
  | |       | +-- center-align@0.1.3
  | |       | | +-- align-text@0.1.4
  | |       | | | `-- longest@1.0.1
  | |       | | `-- lazy-cache@1.0.4
  | |       | +-- right-align@0.1.3
  | |       | `-- wordwrap@0.0.2
  | |       `-- window-size@0.1.0
  | +-- js-yaml@3.6.1
  | | +-- argparse@1.0.7
  | | | `-- sprintf-js@1.0.3
  | | `-- esprima@2.7.3
  | `-- once@1.4.0
  |   `-- wrappy@1.0.2
  +-- istanbul-lib-coverage@1.0.0
  +-- istanbul-lib-instrument@1.1.3
  +-- jest-changed-files@15.0.0
  +-- jest-config@15.1.1
  | +-- istanbul@0.4.5
  | | +-- abbrev@1.0.9
  | | +-- escodegen@1.8.1
  | | | +-- estraverse@1.9.3
  | | | +-- optionator@0.8.2
  | | | | +-- deep-is@0.1.3
  | | | | +-- fast-levenshtein@2.0.4
  | | | | +-- levn@0.3.0
  | | | | +-- prelude-ls@1.1.2
  | | | | +-- type-check@0.3.2
  | | | | `-- wordwrap@1.0.0
  | | | `-- source-map@0.2.0
  | | +-- nopt@3.0.6
  | | +-- supports-color@3.1.2
  | | `-- wordwrap@1.0.0
  | `-- jest-environment-node@15.1.1
  +-- jest-environment-jsdom@15.1.1
  | `-- jsdom@9.5.0
  |   +-- abab@1.0.3
  |   +-- acorn@2.7.0
  |   +-- acorn-globals@1.0.9
  |   +-- array-equal@1.0.0
  |   +-- cssom@0.3.1
  |   +-- cssstyle@0.2.37
  |   +-- iconv-lite@0.4.13
  |   +-- nwmatcher@1.3.8
  |   +-- parse5@1.5.1
  |   +-- request@2.75.0
  |   | +-- aws-sign2@0.6.0
  |   | +-- aws4@1.4.1
  |   | +-- bl@1.1.2
  |   | | `-- readable-stream@2.0.6
  |   | |   +-- core-util-is@1.0.2
  |   | |   +-- isarray@1.0.0
  |   | |   +-- process-nextick-args@1.0.7
  |   | |   +-- string_decoder@0.10.31
  |   | |   `-- util-deprecate@1.0.2
  |   | +-- caseless@0.11.0
  |   | +-- combined-stream@1.0.5
  |   | | `-- delayed-stream@1.0.0
  |   | +-- extend@3.0.0
  |   | +-- forever-agent@0.6.1
  |   | +-- form-data@2.0.0
  |   | | `-- asynckit@0.4.0
  |   | +-- har-validator@2.0.6
  |   | | +-- commander@2.9.0
  |   | | | `-- graceful-readlink@1.0.1
  |   | | `-- is-my-json-valid@2.14.0
  |   | |   +-- generate-function@2.0.0
  |   | |   +-- generate-object-property@1.2.0
  |   | |   | `-- is-property@1.0.2
  |   | |   `-- jsonpointer@2.0.0
  |   | +-- hawk@3.1.3
  |   | | +-- boom@2.10.1
  |   | | +-- cryptiles@2.0.5
  |   | | +-- hoek@2.16.3
  |   | | `-- sntp@1.0.9
  |   | +-- http-signature@1.1.1
  |   | | +-- assert-plus@0.2.0
  |   | | +-- jsprim@1.3.1
  |   | | | +-- extsprintf@1.0.2
  |   | | | +-- json-schema@0.2.3
  |   | | | `-- verror@1.3.6
  |   | | `-- sshpk@1.10.0
  |   | |   +-- asn1@0.2.3
  |   | |   +-- assert-plus@1.0.0
  |   | |   +-- bcrypt-pbkdf@1.0.0
  |   | |   | `-- tweetnacl@0.14.3
  |   | |   +-- dashdash@1.14.0
  |   | |   | `-- assert-plus@1.0.0
  |   | |   +-- ecc-jsbn@0.1.1
  |   | |   +-- getpass@0.1.6
  |   | |   | `-- assert-plus@1.0.0
  |   | |   +-- jodid25519@1.0.2
  |   | |   +-- jsbn@0.1.0
  |   | |   `-- tweetnacl@0.13.3
  |   | +-- is-typedarray@1.0.0
  |   | +-- isstream@0.1.2
  |   | +-- json-stringify-safe@5.0.1
  |   | +-- mime-types@2.1.12
  |   | | `-- mime-db@1.24.0
  |   | +-- node-uuid@1.4.7
  |   | +-- oauth-sign@0.8.2
  |   | +-- qs@6.2.1
  |   | +-- stringstream@0.0.5
  |   | `-- tunnel-agent@0.4.3
  |   +-- sax@1.2.1
  |   +-- symbol-tree@3.1.4
  |   +-- tough-cookie@2.3.1
  |   +-- webidl-conversions@3.0.1
  |   +-- whatwg-url@3.0.0
  |   | `-- tr46@0.0.3
  |   `-- xml-name-validator@2.0.1
  +-- jest-file-exists@15.0.0
  +-- jest-haste-map@15.0.1
  | +-- fb-watchman@1.9.0
  | | `-- bser@1.0.2
  | |   `-- node-int64@0.4.0
  | `-- multimatch@2.1.0
  |   +-- array-differ@1.0.0
  |   +-- array-union@1.0.2
  |   | `-- array-uniq@1.0.3
  |   `-- arrify@1.0.1
  +-- jest-jasmine2@15.1.1
  | +-- jasmine-check@0.1.5
  | | `-- testcheck@0.1.4
  | `-- jest-matchers@15.1.1
  |   `-- jest-matcher-utils@15.1.0
  +-- jest-mock@15.0.0
  +-- jest-resolve@15.0.1
  | +-- browser-resolve@1.11.2
  | `-- resolve@1.1.7
  +-- jest-resolve-dependencies@15.0.1
  +-- jest-runtime@15.1.1
  | +-- babel-jest@15.0.0
  | | `-- babel-preset-jest@15.0.0
  | |   `-- babel-plugin-jest-hoist@15.0.0
  | `-- babel-plugin-istanbul@2.0.1
  |   +-- find-up@1.1.2
  |   | +-- path-exists@2.1.0
  |   | `-- pinkie-promise@2.0.1
  |   |   `-- pinkie@2.0.4
  |   `-- test-exclude@2.1.2
  |     `-- micromatch@2.3.11
  |       +-- arr-diff@2.0.0
  |       | `-- arr-flatten@1.0.1
  |       +-- array-unique@0.2.1
  |       +-- braces@1.8.5
  |       | +-- expand-range@1.8.2
  |       | | `-- fill-range@2.2.3
  |       | |   +-- is-number@2.1.0
  |       | |   +-- isobject@2.1.0
  |       | |   +-- randomatic@1.1.5
  |       | |   `-- repeat-string@1.5.4
  |       | +-- preserve@0.2.0
  |       | `-- repeat-element@1.1.2
  |       +-- expand-brackets@0.1.5
  |       | `-- is-posix-bracket@0.1.1
  |       +-- extglob@0.3.2
  |       +-- filename-regex@2.0.0
  |       +-- is-extglob@1.0.0
  |       +-- is-glob@2.0.1
  |       +-- kind-of@3.0.4
  |       | `-- is-buffer@1.1.4
  |       +-- normalize-path@2.0.1
  |       +-- object.omit@2.0.0
  |       | +-- for-own@0.1.4
  |       | | `-- for-in@0.1.6
  |       | `-- is-extendable@0.1.1
  |       +-- parse-glob@3.0.4
  |       | +-- glob-base@0.3.0
  |       | | `-- glob-parent@2.0.0
  |       | `-- is-dotfile@1.0.2
  |       `-- regex-cache@0.4.3
  |         +-- is-equal-shallow@0.1.3
  |         `-- is-primitive@2.0.0
  +-- jest-snapshot@15.1.1
  | +-- jest-diff@15.1.0
  | `-- pretty-format@3.8.0
  +-- jest-util@15.1.1
  | `-- diff@3.0.0
  +-- json-stable-stringify@1.0.1
  | `-- jsonify@0.0.0
  +-- node-notifier@4.6.1
  | +-- cli-usage@0.1.4
  | | +-- marked@0.3.6
  | | `-- marked-terminal@1.6.2
  | |   +-- cardinal@1.0.0
  | |   | +-- ansicolors@0.2.1
  | |   | `-- redeyed@1.0.0
  | |   +-- cli-table@0.3.1
  | |   | `-- colors@1.0.3
  | |   `-- node-emoji@1.4.1
  | |     `-- string.prototype.codepointat@0.2.0
  | +-- growly@1.3.0
  | +-- lodash.clonedeep@3.0.2
  | | +-- lodash._baseclone@3.3.0
  | | | +-- lodash._arraycopy@3.0.0
  | | | +-- lodash._arrayeach@3.0.0
  | | | +-- lodash._baseassign@3.2.0
  | | | | `-- lodash._basecopy@3.0.1
  | | | +-- lodash._basefor@3.0.3
  | | | +-- lodash.isarray@3.0.4
  | | | `-- lodash.keys@3.1.2
  | | |   +-- lodash._getnative@3.9.1
  | | |   `-- lodash.isarguments@3.1.0
  | | `-- lodash._bindcallback@3.0.1
  | +-- minimist@1.2.0
  | +-- semver@5.3.0
  | `-- shellwords@0.1.0
  +-- sane@1.4.1
  | +-- exec-sh@0.2.0
  | | `-- merge@1.2.0
  | +-- walker@1.0.7
  | | `-- makeerror@1.0.11
  | |   `-- tmpl@1.0.4
  | `-- watch@0.10.0
  +-- which@1.2.11
  | `-- isexe@1.1.2
  +-- worker-farm@1.3.1
  | +-- errno@0.1.4
  | | `-- prr@0.0.0
  | `-- xtend@4.0.1
  `-- yargs@5.0.0
    +-- cliui@3.2.0
    | `-- wrap-ansi@2.0.0
    +-- decamelize@1.2.0
    +-- get-caller-file@1.0.2
    +-- lodash.assign@4.2.0
    +-- os-locale@1.4.0
    | `-- lcid@1.0.0
    |   `-- invert-kv@1.0.0
    +-- read-pkg-up@1.0.1
    | `-- read-pkg@1.1.0
    |   +-- load-json-file@1.1.0
    |   | +-- parse-json@2.2.0
    |   | | `-- error-ex@1.3.0
    |   | |   `-- is-arrayish@0.2.1
    |   | +-- pify@2.3.0
    |   | `-- strip-bom@2.0.0
    |   |   `-- is-utf8@0.2.1
    |   +-- normalize-package-data@2.3.5
    |   | +-- hosted-git-info@2.1.5
    |   | +-- is-builtin-module@1.0.0
    |   | | `-- builtin-modules@1.1.1
    |   | `-- validate-npm-package-license@3.0.1
    |   |   +-- spdx-correct@1.0.2
    |   |   | `-- spdx-license-ids@1.2.2
    |   |   `-- spdx-expression-parse@1.0.3
    |   `-- path-type@1.1.0
    +-- require-directory@2.1.1
    +-- require-main-filename@1.0.1
    +-- set-blocking@2.0.0
    +-- string-width@1.0.2
    | +-- code-point-at@1.0.0
    | | `-- number-is-nan@1.0.0
    | `-- is-fullwidth-code-point@1.0.0
    +-- which-module@1.0.0
    +-- window-size@0.2.0
    +-- y18n@3.2.1
    `-- yargs-parser@3.2.0
      `-- camelcase@3.0.0
```

**Comment by ArtemGovorov** [comment]
Thanks for the details!
I haven't managed to reproduce the same issue, but I have hit another error with the `setupTestFrameworkScriptFile` setting on Windows that may be related to the original one. I have published the fix in core v1.0.297. Could you please restart your IDE, open the sample project, start wallaby (it'll download the latest version), just in case restart wallaby again, and try running it on the sample project to see if the issue has gone?

**Comment by narthollis** [comment]
I went to look at my wallaby config in WebStorm and noted that it was using a bundled node (per the default settings). When I updated this to use my installed node, I managed to get the tests results that I was expecting.

The default path supplied in the WebStorm Wallaby.js config for me was `C:\Users\narthollis\.WebStorm2016.3\system\wallaby\node_v4.2.2\node.exe`

**Comment by narthollis** [comment]
Thanks for taking the time to look into this so quickly.

**Comment by ArtemGovorov** [comment]
Thanks for the update!
Hmm, interesting! Are you sure that it's changing the node version had fixed it and not upgrading to the just published v1.0.297 version? Because you shouldn't need to change the node version in WebStorm wallaby settings - the node version there is used to run wallaby.js core itself, but your tests are running using whatever is specified in the `env.runner`](https://wallabyjs.com/docs/integration/node.html#node-version) setting, which in your case is pointing to your system default node installation.

**Comment by narthollis** [comment]
It would appear that it was infact the 1.0.297. I changed my Wallaby JS config back to the original path and it is still working.

The initial try after restarting WebStorm must have still been running the older wallaby js version.
